### DISCUSSION TITLE
Mod search should prioritize mod name/acronyms over other information

### DISCUSSION THREAD

**Discussion by hrfarmer** [ppy/osu#25979] [Ideas] [answered]
Pretty self-explanatory, I don't think it makes any sense for the game to select the Easy mod when I type hr lmao

https://github.com/ppy/osu/assets/58487401/e7c69e20-3999-435c-a0f0-355cef5fa4c8

**Comment by bdach** [comment, marked as answer]
the reason it matched is that the description is:

> Larger circles, more forgiving HP drain, less accuracy required, and t**_hr_**ee lives!

i'd be honestly fine with excluding description as a search term, because attempting to magically what you might have meant is not happening.

**Reply by frenzibyte** [reply]
https://github.com/ppy/osu/pull/27192
